legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-core/src/main/java/org/finos/legend/engine/persistence/components/relational/api/ErrorCategory.java

```diff
@@ -23,6 +23,7 @@ public enum ErrorCategory
     FILE_NOT_FOUND("File not found in specified location"),
     UNKNOWN("Unknown error"),
     DUPLICATES("Duplicate rows found"),
+    DUPLICATE_PRIMARY_KEYS("Duplicate primary keys found"),
```

Comment:
Make the error mss bit more intuitive 
"Multiple rows with duplicate primary keys found"